### ISSUE TITLE
Replace hard-coded messages with slot-based customization

### ISSUE THREAD

**Issue by tomalec** [Palindrom/palindrom-client#37]
Currently, we have hard-coded texts, and HTML structure for presenting messages at https://github.com/PuppetJs/puppet-polymer-client/blob/gh-pages/puppet-client.html#L72

Due to i18n, a11y, customization, theming, cheeper maintenance, etc. those should be customizable.

In my opinion the best way to do so is to use `<slot>`, we can provide fallback content inside, but tu CE consumer could simply write his/her own:

```html
<puppet-client>
   <div slot="reconnecting-now" class="my-custom-styles">Łączę ponownie</div>
</puppet-client>
```
So we will no longer have to bother about how to show the message for the mobile, in different language, in different context, etc.

//cc @warpech @Mnosek

**Comment by alshakero** [comment]
Do you think we should still do this for `palindrom-error-catcher`?

**Comment by tomalec** [comment]
I think it could be useful. But not with such priority.

**Comment by warpech** [comment]
I agree with @tomalec, I will move it to some other milestone.

**Comment by warpech** [comment]
Moved to https://github.com/Palindrom/palindrom-error-catcher/issues/2
